examples/node-mssql-programmatic-use/src/sync-db.js

```diff
@@ -1,4 +1,4 @@
-import { synchronize, loadConfig, resolveConnections } from '@leapfrogtechnology/sync-db';
+const { synchronize, loadConfig, resolveConnections } = require('@leapfrogtechnology/sync-db');
```

Comment:
Why require? Can we not use imports?

Reply:
did it to test the programmatic `synchronize()` using node. So, changed it to require